test/unit/vl-tabs.test.html

```diff
@@ -160,4 +183,4 @@
     });
   </script>
```

Comment:
misschien nog een explicitie test toevoegen dat alles ook correct werkt indien er aan het title slot iets veranderd wordt? maw dat die change "doorgepropageerd" wordt.

Reply:
test bijgeschreven die nagaat of het updaten van een title slot doorkomt. En dat was niet het geval. De mutator keek niet naar de elementen binnen de vl-tabs-pane (enkel de rechtstreekse). En er werd ook nog geen rekening gehouden om het gekopiërde slot te vervangen door de nieuwe inhoud (enkel toevoegen was nu het geval).